src/components/Publish/_validation.ts

```diff
@@ -32,7 +32,17 @@ const validationService = {
   files: Yup.array<FileInfo[]>()
     .of(
       Yup.object().shape({
-        url: Yup.string().url('Must be a valid URL.').required('Required'),
+        url: Yup.string()
+          .test(
+            'GoogleNotSupported',
+            'Google drive is not a supported hosting service. Please use an alternative.',
```

Comment:
Good enough, we probably want this on Edit validation too?

```suggestion
            'Google Drive is not a supported hosting service. Please use an alternative.',
```

Reply:
Cool, yeah I'll add it on the edit form as well